packages/smooth_app/lib/pages/personalized_ranking_page.dart

```diff
@@ -30,12 +30,18 @@ class _PersonalizedRankingPageState extends State<PersonalizedRankingPage> {
     SmoothItModel.MATCH_INDEX_NO,
   ];
 
-  static const Map<int, Color> _COLORS = <int, Color>{
-    SmoothItModel.MATCH_INDEX_ALL: null,
-    SmoothItModel.MATCH_INDEX_YES: Colors.green,
-    SmoothItModel.MATCH_INDEX_MAYBE: Colors.grey,
-    SmoothItModel.MATCH_INDEX_NO: Colors.red,
-  };
+  static Map<int, Color> _COLORS = <int, Color>{};
```

Comment:
Not nice, and wrong (every rebuild, you add colors). I understand the reason, though.
My suggestion:
* Put back the whole `static const` syntax but remove the `SmoothItModel.MATCH_INDEX_ALL` entry.
* when you use `_COLORS[index]`, add the "if null" syntax: `color: _COLORS[matchIndex] ?? Theme.of(context).accentColor`